dts/bindings/bluetooth/infineon,cyw43xxx-bt-hci.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    CYW43xxx Connectivity that uses Zephyr's Bluetooth Host Controller Interface UART
+    driver.
+
+compatible: "infineon,cyw43xxx-bt-hci"
+
+include: base.yaml
+
+properties:
+    bt-reg-on-gpios:
+      type: phandle-array
+      required: false
+
+    bt-host-wake-gpios:
+      type: phandle-array
+      required: false
+
+    bt-dev-wake-gpios:
+      type: phandle-array
+      required: false
+
+    baudrate_for_fw_download:
+       type: int
+       default: 3000000
```

Comment:
Default should not be part of the binding, but rather put into the code with `DT_PROP_OR(..., 3000000)`.

Another thing is whether `3000000` should be hardcoded as a speed at all? IMO when `baudrate-for-fw-download` won't be provided, then the default should be `current-speed` of UART node.

Will it be possible to use this BT chip with any host MCU or just Infineon? If any MCU can be used, then 3Mbps might not be supported. So better to default to not switching baudrate for the firmware transfer.

Reply:
it can be use with any MCU, i will update:

1. rename `baudrate-for-fw-download` to `fw-download-speed` as was suggestion by @fabiobaltieri (done)
2. add default DT_PROP_OR(..., 115200) for `fw-download-speed`
3. in board (cy8cproto_062_4343w) configuration will declare  default 3M `fw-download-speed = <3000000>;` as shown in commit [boards: arm: cy8cproto-062-4343w: enable BLE features](https://github.com/zephyrproject-rtos/zephyr/pull/55014/commits/51667b754d0bcff3a9de4519c60a2fff03e47dc9)  (done)